python/paddle/fluid/layers/nn.py

```diff
@@ -9334,6 +9335,76 @@ def get_tensor_from_selected_rows(x, name=None):
     return out
 
 
+def shuffle_channel(x, group, name=None):
+    """
+    **Shuffle Channel Operator**
```

Comment:
```suggestion
    **Shuffle Channel Operator**

```
add a blank line to separate with the main body of the api.